grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -26,30 +27,48 @@
 
```

Comment:
FYI, you can do this if you want the tag to be the name of the class
`private final String TAG = MainActivity.class.getSimpleName();`